core/src/test/java/org/infinispan/lock/singlelock/AbstractInitiatorCrashTest.java

```diff
@@ -30,6 +30,7 @@ public void testInitiatorCrashesBeforeReleasingLock() throws Exception {
       releaseLocksLatch.await(30, TimeUnit.SECONDS);
 
       assert checkTxCount(0, 0, 1);
```

Comment:
could we start considering refactoring these assert statements whenever we touch a test that have them?

Reply:
Sorry, I missed your comment @karesti. We already have a policy of replacing the `assert` keyword with `AssertJUnit` when we touch the test, but honestly I didn't even think about the asserts at the time: the test was failing and I wanted to get the fix in ASAP.